Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+5. **Delete a Seed.** You can also just delete a seed.
+
+### Viewing the Main Menu
+
+<blockquote>
+  <img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-bar.jpg" align="center" width=500>
+</blockquote>
+
+The main menu contains three buttons in the menu bar along the top:
+
+* **Information** (circled "i"). Read documentation on all of the specifications and data types found in **Seed Tool**.
+* **Scan** (qr code). Import a seed (see "Importing a Seed") or a `crypto-request` (see "Using a Seed") from a QR code; or import text from the Clipboard.
+* **Setting** (gear). Choose MainNet or TestNet as your default network (this is used for key derivation, as discussed in "Deriving a Key"); choose whether to sync to iCloud or not. If you wanted to erase all of your data, this would be the place to do so.
+
+> :warning: **WARNING:** We highly suggest you leave iCloud backups on. Without it, if you lose your phone, you will lose all of your seeds. The iCloud backups are encrypted, so no one but you should be able to acces them.
+
+Under the main menu are options to **add** ("+") and **delete** ("edit") seeds followed by a list of each of your seeds, with each seed identified by an Object identity Block ("OID"). You can click the right arrow on a seed to see more data about it and to export it.
+
+> :warning: **WARNING:** Deleting your seeds through the Settings or edit functions will entirely remove them: they will be gone!
+
+## Adding a Seed
+
+Seeds can be created or imported. Seed Creation is done through the **add** ("+") button, while seed importing is done via either the qr button (which imports via camera or other automated means) or the **add** button (which does so via cut-and-pasting).
+
+### Scanning a Seed via Automated Means
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-scan.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-scan.jpeg" align="right" width=250></a>
+
+The **Scan** (qr code) button on the main menu provides the most automated methods for importing seeds, using your camera, the cut-and-paste clipboard, the file system, or your photo roll. 
+
+To scan a QR code, you can either point your camera at the QR of a seed, import a QR of a seed from your Photos, or choose an image of a QR from your Files. 
+
+To scan text defining a seed, just copy it into your Clipboard and click Paste. 
+
+Note that for these methodologies, **Seed Tool** expects the QR code or the clipboard to contain a [Uniform Resource](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md), a standardized way to encode data in an efficient and self-identifying way. This will usually mean a [`ur-crypto-seed`](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-006-urtypes.md#cryptographic-seed-crypto-seed).
```

Comment:
We may want to mention somewhere, maybe in the threat model, that having seeds or key material on the clipboard is a minor risk — potential other apps on the device can see the clipboard, and if if you have iCloud's handoff feature enabled, it will sync to your other devices. To mitigate this somewhat, we do clear the clipboard after X seconds — imperfect, but the convenience of being able to leverage the clipboard is important.

Reply:
(1) Threat description added to the Threats section.
(2) ur:crypto-seed corrected